test/fixtures/network-raft.json

```diff
@@ -28,6 +28,21 @@
         }
       }
     },
+    "orderer5.example.com": {
```

Comment:
the naming here is a bit confusing, why starting from `orderer5`?

Reply:
There are 5 orderers and I have added only the credentials of `orderer` and `orderer5` in the `network.json` file.